src/lib/components/EventsPanel.svelte

```diff
@@ -23,17 +24,14 @@ import Card from './Card.svelte';
 		{#each events as event}
 			<div class="flex items-center p-2 border rounded-lg">
 				<CalendlyPopupButton bookingLink={event.bookingLink}>Book</CalendlyPopupButton>
-				<!-- <div class="w-16 h-16 bg-gray-100 rounded-lg overflow-hidden">
-					<img src={event.imageSrc} alt={event.title} class="object-cover w-full h-full" />
-				</div> -->
 				<div class="ml-4 flex-grow">
 					<h3 class="font-semibold">{event.title}</h3>
 					<p class="text-sm text-gray-600 truncate">{event.description}</p>
 				</div>
-				<div class="ml-4">
-					<span class="block text-sm bg-gray-100 py-2 px-4 rounded" aria-label="Event Date">
+				<div class="ml-4 min-w-fit">
+					<span class="block text-sm bg-gray-100 py-2 px-4 rounded" aria-label="Event Duration">
 						<img class="mx-auto" src="$lib/assets/design/icons/calendar.svg" alt="" aria-hidden />
-						{event.date}
+						{event.duration}
```

Comment:
One thing to note:

I changed the "date" part of the events panel to just a duration... The way Calendly works is that if no one has signed up for an event it isn't marked as Scheduled yet, and there's no like "next available date" field on an event type since i guess people can schedule any time. Not sure if we want the duration there or something else, or to take it out entirely.

Reply:
Okay interesting. I think it looks fine in the deploy preview, but we may want to change the icon from a calendar to a clock

Reply:
Functionality-wise we'll see what they say

Reply:
Does this endpoint not work for fetching times? Is it empty if nobody has booked it?
https://developer.calendly.com/api-docs/6a1be82aef359-list-event-type-available-times

Reply:
yeah i thought about using this but it only goes 7 days out so if they have an event more than a week away it won't show... but it's better than nothing i suppose ! ill check it out tomorrow

Reply:
Honestly I just removed the date thing for now, we can see what they want and figure it out from there!